src/browser/base/content/zen-styles/zen-tabs/vertical-tabs.css

```diff
@@ -7,6 +7,10 @@
     height: 100%;
   }
 
+  #TabsToolbar {
+    padding: var(--zen-toolbox-padding);
```

Comment:
Not sure about this, itll make the tabs be un-aligned

Reply:
It actually aligns all tabs and makes sure they all share common padding. Otherwise every tab must have its margin set, and they were not consistent between different modes.

Also setting it as `--zen-toolbox-padding` will allow additional consistency.

Then only difference is between expandable & non-expandable as the top border adds 1px padding.
See:
![zen_UMlIAv9Bfa](https://github.com/user-attachments/assets/3465c9a2-15e3-4cda-9eb8-5bd0ddf27173)